crypto/signature.go

```diff
@@ -261,12 +261,22 @@ func verifySignature(
 				return time.Unix(verifyTime, 0)
 			}
 
+			seeker, ok := origText.(io.ReadSeeker)
+			if !ok {
+				return nil, newSignatureFailed(errors.New("message reader do not support seeking, cannot retry signature verification"))
```

Comment:
```suggestion
				return nil, newSignatureFailed(errors.Wrap(err, "gopenpgp: message reader do not support seeking, cannot retry signature verification"))
```
We should prefix the error message with the module name. And I would include the original error.

Reply:
Fixed!